frontend/composables/notifications/useNotificationsManagementDashboards.ts

```diff
@@ -126,7 +126,7 @@ export function useNotificationsManagementDashboards() {
           dashboard_id,
           group_id,
         },
-      )
+      ).then(() => refreshOverview())
```

Comment:
this wont work as you are returning on line `120`

Reply:
ahh of course , thx :pray: